adaptors/backbone/backbone_view_widget.js

```diff
@@ -93,6 +97,26 @@ BackboneViewWidget.prototype.attach = function attach(elem) {
     parentView: this.parentView,
     template: this.template
   });
+  this.isInitialized = true;
+  // If callbacks were registered for postInitialization, process them
+  if (this._isInitializedCallbacks) {
+    if (this.view.isInitialized) {
+      this._isInitializedCallbacks.forEach((fn) => fn());
+    } else {
+      this.view.once('initialized', () => {
+        this._isInitializedCallbacks.forEach((fn) => fn());
+        this._isInitializedCallbacks.length = 0;
+      });
+    }
+  }
+};
+
+/**
+ * Binds a function to process after this view's postInitialize method
+ * @param  {Function} callback
+ */
+BackboneViewWidget.prototype.waitForAttach = function(callback) {
+  this._isInitializedCallbacks.push(callback);
```

Comment:
these will only be bound in the attach function and mimic the current functionality: https://github.com/wayfair/tungstenjs/blob/master/adaptors/backbone/base_view.js#L377
This is actually a bit better since that's a straight `listenTo` rather than `listenToOnce`